tool-box/Dockerfile

```diff
@@ -1,16 +1,32 @@
-FROM fedora:latest
+FROM registry.access.redhat.com/ubi8
+
+MAINTAINER Aly Ibrahim<aly@redhat.com>
 
 ENV OC_CLIENT_MIRROR https://mirror.openshift.com/pub/openshift-v3/clients/3.11.127/linux/oc.tar.gz
 ENV HOME /home/tool-box
 
-RUN dnf -y update && \
-    INSTALL_PKGS="git tree vim unzip python-pip jq" && \
-    dnf -y install $INSTALL_PKGS && \
-    dnf clean all && \
-    curl $OC_CLIENT_MIRROR | tar -C /usr/local/bin/ -xzf - && \
-    mkdir -m 775 $HOME && \
-    chmod 775 /etc/passwd && \
-    pip install git+https://github.com/ansible/ansible.git@stable-2.6
+RUN yum -y update && \
+    INSTALL_PKGS="git vim unzip python36" && \
+    yum -y install $INSTALL_PKGS && \
+    yum clean all
+
+RUN curl -o jq -sL https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 && \
+    chmod +x jq && \
+    mv jq /usr/local/bin
+
+RUN mkdir -m 775 $HOME && \
+    chmod 775 /etc/passwd && \ 
+    pip3 install git+https://github.com/ansible/ansible.git@stable-2.8
+
+RUN  curl -s https://storage.googleapis.com/kubernetes-helm/helm-v2.9.0-linux-amd64.tar.gz | tar -xvz && \
+     chmod u+x linux-amd64/helm && mv linux-amd64/helm /usr/local/bin/ && rm -rf linux-amd64 && \
+     helm init --client-only
```

Comment:
Can we use the latest stable relsease: https://github.com/helm/helm/releases/latest?
And a nitpick, there are two spaces between `RUN` and the commands.

Reply:
I changed to 2.14.3, latest stable currently.
Extra space removed.

Reply:
Thank you @AlyIbrahim, but there's still an extra space before `helm init`

Reply:
@pabrahamsson Done ;)